src/main/java/practice/StreamPractice.java

```diff
@@ -75,6 +107,10 @@ public List<String> getCatsNames(List<Person> peopleList, int femaleAge) {
      * parametrized with Candidate in CandidateValidator.
      */
     public List<String> validateCandidates(List<Candidate> candidates) {
-        return Collections.emptyList();
+        return candidates.stream()
+                .filter(c -> new CandidateValidator().test(c))
+                .map(Candidate::getName)
+                .sorted(String::compareTo)
```

Comment:
Default usage will be enough to implement well 
```suggestion
                .sorted()
```